pkgs/development/libraries/agda/cubical/default.nix

```diff
@@ -11,15 +11,15 @@ mkDerivation rec {
     hash = "sha256-bnHz5uZXZnn1Zd36tq/veA4yT7dhJ1c+AYpgdDfSRzE=";
   };
 
-  LC_ALL = "en_US.UTF-8";
-
-  preConfigure = ''export AGDA_EXEC=agda'';
+  LC_ALL = "C.UTF-8";
 
   # The cubical library has several `Everything.agda` files, which are
   # compiled through the make file they provide.
-  nativeBuildInputs = [ ghc glibcLocales ];
+  nativeBuildInputs = [ ghc ];
   buildPhase = ''
+    runHook preBuild
```

Comment:
Are these needed to get the locale to work?

Reply:
No, unrelated. https://nixos.org/manual/nixpkgs/unstable/#sec-stdenv-phases

Reply:
Is there a reason we need to include this then? Is it just generally a good idea?

Reply:
It's good practice to allow downstream users to override hooks.